curator/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+rpm -q epel-release || yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum install -y --setopt=tsflags=nodocs \
+  python-pip \
+  PyYAML
+pip install elasticsearch-curator python-crontab
+yum clean all
+
+# HACK HACK HACK - remove when fixed upstream
+yum install -y --setopt=tsflags=nodocs \
+    git
+git clone -b issue-344 https://github.com/richm/elasticsearch-py.git
```

Comment:
Yes.  For enterprise we will have to go through the usual approved patch process.  It's going to be a pain anyway, since we're using epel and/or pip packages.